tracks/csharp/exercises/nucleotide-count/mentoring.md

```diff
@@ -0,0 +1,57 @@
+### Reasonable solutions
+
+### Using a foreach loop
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class NucleotideCount
+{
+    public static IDictionary<char, int> Count(string sequence)
+    {
+        var result = new Dictionary<char, int> { {'A', 0}, {'C', 0}, {'G', 0}, {'T', 0} };
+        
+        foreach (var c in sequence)
+        {
+            if (result.ContainsKey(c)) result[c]++;
+            else throw new ArgumentException("Invalid Nucleotide Symbol");
+        }
+
+        return result;
+    }
+}
+```
+
+### Foreach with ToDictionary
```

Comment:
```suggestion
#### Foreach with ToDictionary
```

Reply:
Resolved in 3bf9ce67cbd9ef4eef08e3718ec54ef4e11ed10c